.gitignore

```diff
@@ -116,3 +116,5 @@ test/cache/*
 
 libbitcoinconsensus.pc
 contrib/devtools/split-debug.sh
+
+.idea
```

Comment:
Remove this. [See "Ignoring IDE/editor files" in the developer notes](https://github.com/bitcoin/bitcoin/blob/f504a1402afd0760e9d348ecc8bad0094aa7d705/doc/developer-notes.md#ignoring-ideeditor-files) :-)